server/game/cards/02.3-EDS/AngelaPayne.js

```diff
@@ -0,0 +1,20 @@
+const DudeCard = require('../../dudecard.js');
+
+class AngelaPayne extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            ifCondition: () => this.locationCard.owner !== this.controller && this.locationCard.getType() === 'deed',
+            ifFailMessage: context => this.game.addMessage('{0} boots {1} but does not gain any ghostrock because she is not at a deed {0} doesn\'t own', context.player, this),
+            message: context => this.game.addMessage('{0} boots {1} to gain 2 ghostrock', context.player, this),
```

Comment:
We should use GR from now on so it is correctly substituted by a bullet icon in chat.
```suggestion
            message: context => this.game.addMessage('{0} boots {1} to gain 2 GR', context.player, this),
```